training/movinet_with_hmdb51/AVideoClips.py

```diff
@@ -0,0 +1,149 @@
+from typing import Any, Dict, List, Optional, Tuple
+from torchvision.datasets.video_utils import VideoClips
+from torchvision.datasets.video_utils import read_video_timestamps
+from torchvision.io.video import read_video
+import tempfile
+import os
+import shutil
+from torch.utils.data.dataloader import DataLoader
+import torch
+
+from aperturedb.Videos import Videos
+from tqdm import tqdm
+
+class _VideoTimestampsDataset:
+    """
+    Dataset used to parallelize the reading of the timestamps
+    of a list of videos, given their paths in the filesystem.
+
```

Comment:
The comment is not explaining much. but you did say you plan to add more comments. so that's good. if we need a class hierarchy diagram, maybe that helps